internal/handlers/handler_test.go

```diff
@@ -468,3 +468,82 @@ func TestFind(t *testing.T) {
 		})
 	}
 }
+
+func TestCount(t *testing.T) {
+	t.Parallel()
+
+	ctx := testutil.Ctx(t)
+	pool := testutil.Pool(ctx, t)
+	l := zaptest.NewLogger(t)
+	shared := shared.NewHandler(pool, "127.0.0.1:12345")
+	sql := sql.NewStorage(pool, l.Sugar())
+	jsonb1 := jsonb1.NewStorage(pool, l)
+	handler := New(pool, l, shared, sql, jsonb1)
```

Comment:
It uses options structure now: https://github.com/FerretDB/FerretDB/blob/c52ea450072bd3453f27146c49a6869a8a7c25fb/internal/handlers/handler_test.go#L42-L49